constants.go

```diff
@@ -684,15 +683,16 @@ const (
 	SystemAccessApproverUserName = "@teleport-access-approval-bot"
 )
 
-// MinClientVersion is the minimum client version required by the server.
-var MinClientVersion string
-
-func init() {
+var (
+	// SemVersion is the Version represented as a [semver.Version].
+	SemVersion = semver.New(Version)
```

Comment:
Why do we have a separate `version.go` for the main module instead of just `const Version = api.Version` (and `var SemVersion = api.SemVersion`)?